teamcity/flake8_plugin.py

```diff
@@ -1,74 +1,28 @@
-try:
-    import pycodestyle as pep8
-except ImportError:
-    import pep8
+from flake8.formatting import base
 
 import re
 
 from teamcity.messages import TeamcityServiceMessages
-from teamcity import __version__, is_running_under_teamcity
```

Comment:
I guess maybe in `tox.ini`, one could put:

```
commands = flake8 {env:FLAKE8_OPTIONS:}
```

And then set the `FLAKE8_OPTIONS` environment variable in the TeamCity build config. 

Any other ideas?